docker/arm-server/Dockerfile

```diff
@@ -1,30 +1,53 @@
-FROM ubuntu:22.04
-
+# Docker image for running CF instances in ARM64 server.
+# Docker image includes HO(Host Orchestrator) inside,
+# so it could execute CF instance with API in HO.
+FROM ubuntu:22.04 AS cuttlefish-arm64
+
+# Expose Operator Port (HTTP:1080, HTTPS:1443)
+EXPOSE 1080 1443
+# Expose HO(Host Orchestrator) Port (HTTP:2080, HTTPS:2443)
+EXPOSE 2080 2443
+# Expose WebRTC Port
 EXPOSE 15550-15560
-EXPOSE 1443
+# Expose ADB Port
+# Corresponding ADB port for CF instance is, 6520+instance_num-1.
 EXPOSE 6520-6620
+
 USER root
 
 RUN set -x
 
+# Install dependants of CF debian packages.
 RUN apt update
-RUN apt install -y --no-install-recommends git
-RUN apt install -y --no-install-recommends ca-certificates && update-ca-certificates
-RUN apt install -y --no-install-recommends dpkg-dev build-essential:native config-package-dev debhelper-compat golang-1.13 libprotobuf-dev protobuf-compiler curl sudo
-RUN apt install -y --no-install-recommends gcc-10
-
+RUN apt install -y --no-install-recommends \
```

Comment:
if we decided not to build debian packages here, we need only a part of them.

Reply:
OOC, when you made the initial script, would there be some packages installed for other purposes?

At least for me, `curl` and `sudo` would be useful. Do you have any packages want to make remained?

Reply:
no, i just installed them to build deb packages.

Reply:
Okay, I got that. I'll test with clearing every dependencies except `curl` and `sudo` then.